engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -576,6 +576,11 @@ public void startApplication() throws YarnException, IOException
         vargs.add("-Dlog4j.debug=true");
       }
 
+      String loggerAppender = dag.getValue(LogicalPlan.LOGGER_APPENDER);
```

Comment:
@vrozov I support having the ability to have separate configuration for cli and asked for it. What I am suggesting is for having an easy way to specify configuration for both when they are the same.